src/libcollectionstest/enum_set.rs

```diff
@@ -60,9 +62,8 @@ fn test_len() {
     assert_eq!(e.len(), 0);
 }
 
-///////////////////////////////////////////////////////////////////////////
-// intersect
-
+/// ////////////////////////////////////////////////////////////////////////
```

Comment:
Could you manually fixup these comment changes please (to undo what rustfmt has done)